src/lib.rs

```diff
@@ -3,3 +3,35 @@ pub mod models;
 pub mod schema;
 pub mod seed;
 pub mod utils;
+
```

Comment:
This is just an example of what tests should look like. Unit tests and integration tests will likely require some external dev database conn

Reply:
Can't we provision a db in-memory (SQLite?), migrate and seed it in a beforeAll/Each and then tear it down afterwards? Seems better to me than maintaining and pairing for a testing database.

Reply:
Also I don't see any tests in the PR?

Reply:
Like this: https://levelup.gitconnected.com/using-sqlite-with-rust-and-actix-web-with-tests-11a935ac3d95